cordova-lib/src/cordova/plugin.js

```diff
@@ -238,7 +239,21 @@ module.exports = function plugin(command, targets, opts) {
                 return opts.plugins.reduce(function(soFar, target) {
                     // Check if we have the plugin.
                     if (plugins.indexOf(target) < 0) {
-                        return Q.reject(new CordovaError('Plugin "' + target + '" is not present in the project. See `'+cordova_util.binname+' plugin list`.'));
+                        // Convert target from package-name to package-id if necessary
+                        var keys = Object.keys(pluginMapper);
+                        //Traverse through pluginMapper values to see if it equals our target.
+                        //Cordova-plugin-device would get changes to org.apache.cordova.device
+                        for (var i = 0; i < keys.length; i++) {
```

Comment:
I don't know why we are iterating anyway?  Its already old_id->new_id.  Can't we just `.hasOwnProperty()` ?